docs/control-ssh.adoc

```diff
@@ -0,0 +1,5 @@
+= SSH control
+
+Connects to the target machine via SSH.
+The URI of the form `ssh://user@host:port` is required.
```

Comment:
This is not true at the moment. Only `ssh://host[:port]` is supported, that is why `username` is an additional key to be populated. It definitely should be addressed though.